admin-dev/themes/default/template/helpers/options/options.tpl

```diff
@@ -131,7 +131,7 @@
 									{if $field['type'] == 'select'}
 										<div class="col-lg-9">
```

Comment:
Hi @PululuK. 
`isset($field['multiple']) && $field['multiple']` can be replaced by `!empty($field['multiple'])`.
According to php.net: "A variable is considered empty if it does not exist or if its value equals false"

Reply:
Thanks @lartist